package.json

```diff
@@ -5,13 +5,17 @@
   "description": "-- a listing of things useful to the Solid Community",
   "main": "catalog.html",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "rdf-transform-cli ./v3/catalog-shacl.shce ./v3/catalog-shacl.ttl -p"
```

Comment:
@jeff-zucker you can use this script to generate turtle

Reply:
It uses https://www.npmjs.com/package/rdf-transform-cli which you can see as `devDependency` in `package.json`

Reply:
> It uses https://www.npmjs.com/package/rdf-transform-cli

Which uses a java tool chain and I still would like to know why it is preferable to Jesse's shaclc-parser?

 >which you can see as `devDependency` few lines below

"A few lines below in what package.json?"  I still do not know where you are talking about running this build.

Reply:
Yup exactly; `npm i && npm run build` is all you need to do to generate the turtle.